_topics/en/swap-in-potentiam.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Swap-in Potentiam (SIP)
+
+## Optional. test Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: swap-in potentiam
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Swap-in-Potentiam (SIP)** is a protocol designed to enable the immediate transfer
+  of already-confirmed onchain Bitcoin funds to the Lightning Network, lowering
+  trust requirements by precommitting to an LSP and incorporating a timeout
+  mechanism for unilateral access.
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+  - title: Moving Onchain Funds "Instantly" To Lightning (Jesse Posner, ZmnSCPxj)
+    link: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-January/003810.html
+
+  - title: LSPS C= Extension, Swap-in-Potentiam
+    link: https://github.com/ZmnSCPxj-jr/swap-in-potentiam/blob/master/doc/swap-in-potentiam.md
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "Swap-in Potentiam Announced"
+    url: /en/newsletters/2023/12/20/#swap-in-potentiam
+
+  - title: Non-interactive LN channel open commitments
+    url: /en/newsletters/2023/01/11/#non-interactive-ln-channel-open-commitments
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Hash Time Locked Contracts from Bitcoin Wiki
+    link: https://en.bitcoin.it/wiki/Hash_Time_Locked_Contracts
+
+  - title: Swap-in potentiam for Phoenix
+    link: https://gist.github.com/t-bast/5fd89979a8088b99d0b95c124902aa56
+
+---
+SIP is a [submarine swap][topic submarine swaps]
```

Comment:
Is it?  As I think about it, a submarine swap starts with Alice and Bob both already having channels; Alice creates a UTXO that Bob can only spend if by revealing a preimage that will allow Alice to claim sats in her channel.

A SIP can start with neither party having a channel and only one party contributing any funding.  The second party doesn't give the first party anything (except a signature, if the first party wants to fully open the channel).  That doesn't seem like a "swap" to me at all.  I think it's just an alternative channel opening protocol.

Reply:
Mermaidjs is not supported.  You can draw graphs in any tool, export or screenshot them, and include a PNG.  Several articles do this, you can `cd _topics/en/ ; git grep '\.png'` for examples.

I still don't think SIPs are a type of submarine swap, but if that's a claim supported by its authors, I'm fine with us saying that.

Reply:
I am also surprised by the categorization of a swap-in potentiam as a Submarine Swap. Maybe the definition of Submarine Swaps has evolved, but my understanding of Submarine Swaps was that they allowed the first or the last hop of a multi-hop payment to be executed on-chain, and it involved the on-chain funds to be locked in an HTLC construction.

Reply:
I still don’t see how it would be beneficial to group these together. 

FWIU, SIP refers to receiving an on-chain payment in a way that stages funds for a later splice-in or channel opening operation because they are already temporarily co-owned by the channel co-owners (or to-be channel co-owners). I would be open to an argument that splicing is similar to submarine swaps, but SIP seems to be a separate, removed step from it.